ui/components/app/permissions-connect-permission-list/permissions-connect-permission-list.js

```diff
@@ -18,10 +17,6 @@ import { CaveatTypes } from '../../../../shared/constants/permissions';
  * @returns {JSX.Element} A permission description node.
  */
 function getDescriptionNode(permission, index, accounts) {
-  const permissionValue = permission?.permissionValue?.caveats?.find(
```

Comment:
This seems to revert https://github.com/MetaMask/metamask-extension/commit/4fa502b0d91606513518e32c790d76c7d65aae94 - is that intentional?

Reply:
Yes, because caveats doesn't exist in permissionValue object

Reply:
Hmm, any idea why this value changed since that fix then? 🤔